client/src/pages/Login/Login.tsx

```diff
@@ -40,32 +38,23 @@ export default function Login(): JSX.Element {
   };
 
   return (
-    <div>
+    <>
       <LandingNavbar />
-      <Grid container component="main" className={classes.root}>
+      <Grid container component="main" className={classes.root} justify="center">
         <CssBaseline />
-        <Grid item xs={12} elevation={6} component={Paper} square>
-          <Box className={classes.authWrapper}>
-            <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-              <Grid container>
-                <Grid item xs>
-                  <Typography className={classes.welcome} component="h1" variant="h5">
-                    Welcome back!
-                  </Typography>
-                </Grid>
+        <Grid item xs={12} sm={8} md={6} elevation={6} component={Paper} square>
+          <Box width="100%" maxWidth={350} p={3} marginY={0} marginX="auto">
```

Comment:
You can consolidate `marginY` and `marginX` into `margin="auto"` and it will have the same style and be more concise in the code.

Reply:
I will do that.